src/Transaction/TransactionException.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace PhpInPractice\Matters\Aggregate\Transaction;
+
+class TransactionException extends \RuntimeException
+{
+
+}
```

Comment:
It is a little bit; but I wanted to provide third-parties with a common base to work with